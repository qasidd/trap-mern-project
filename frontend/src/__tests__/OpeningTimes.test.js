import OpeningTime from '../components/Opening-Times/CinemaTime';
import '@testing-library/jest-dom';
import { expect, it } from '@jest/globals';
import { create } from 'react-test-renderer';

it(`It should match the snapshot every time it's rendered`, () => {
    const instance = create(<OpeningTime/>).toJSON();
    expect(instance).toMatchSnapshot();
    
})