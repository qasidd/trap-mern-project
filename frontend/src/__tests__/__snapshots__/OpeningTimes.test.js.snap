// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should match the snapshot every time it's rendered 1`] = `
Array [
  <div
    className="Container text-center"
  >
    <h3>
       Our Opening Times
    </h3>
  </div>,
  <br />,
  <table
    className="table table-striped table-bordered table-hover"
  >
    <thead>
      <tr
        className=" container text-center"
      >
        <th>
          #
        </th>
        <th
          colSpan="1"
        >
          Monday
        </th>
        <th
          colSpan="1"
        >
          Tuesday
        </th>
        <th
          colSpan="1"
        >
          Wednesday
        </th>
        <th
          colSpan="1"
        >
          Thursday
        </th>
        <th
          colSpan="1"
        >
          Friday
        </th>
        <th
          colSpan="1"
        >
          Saturday
        </th>
        <th
          colSpan="1"
        >
          Sunday
        </th>
      </tr>
    </thead>
    <tbody>
      <tr
        className="container text-center"
      >
        <td
          colSpan="1"
        >
          Morning
        </td>
        <td>
          10:00-12:30
        </td>
        <td>
          9:30-11:30 
        </td>
        <td>
          10:15-12:15
        </td>
        <td>
          10:00-12:20
        </td>
        <td>
          9:30-12:30
        </td>
        <td>
          10:00-12:00
        </td>
        <td>
          10:00:12-15
        </td>
      </tr>
      <tr
        className="container text-center"
      >
        <td
          colSpan="1"
        >
          Afternoon
        </td>
        <td>
          13:00-17:00
        </td>
        <td>
          12:00-18:00
        </td>
        <td>
          12:45-18:15
        </td>
        <td>
          12:40-17:00
        </td>
        <td>
          13:00-18:30
        </td>
        <td>
          12:30-17:00
        </td>
        <td>
          12:15-16:15
        </td>
      </tr>
      <tr
        className="container text-center"
      >
        <td
          colSpan="1"
        >
          Evening
        </td>
        <td>
          17:25-22:00
        </td>
        <td>
          18:30-23:00
        </td>
        <td>
          18:15-00:15
        </td>
        <td>
          17:30-23:45
        </td>
        <td>
          19:00-01:00
        </td>
        <td>
          17:00-00:30
        </td>
        <td>
          16:45-22:45
        </td>
      </tr>
    </tbody>
  </table>,
  <br />,
  <div
    className="Container text-center"
  >
    <h3>
       Opening Times During Christmas and Easter Holidays
    </h3>
  </div>,
  <br />,
  <table
    className="table table-striped table-bordered table-hover"
  >
    <thead>
      <tr
        className=" container text-center"
      >
        <th>
          #
        </th>
        <th
          colSpan="1"
        >
          Monday
        </th>
        <th
          colSpan="1"
        >
          Tuesday
        </th>
        <th
          colSpan="1"
        >
          Wednesday
        </th>
        <th
          colSpan="1"
        >
          Thursday
        </th>
        <th
          colSpan="1"
        >
          Friday
        </th>
        <th
          colSpan="1"
        >
          Saturday
        </th>
        <th
          colSpan="1"
        >
          Sunday
        </th>
      </tr>
    </thead>
    <tbody>
      <tr
        className="container text-center"
      >
        <td
          colSpan="1"
        >
          Morning
        </td>
        <td>
          -
        </td>
        <td>
          -
        </td>
        <td>
          10:15-12:15
        </td>
        <td>
          10:00-12:20
        </td>
        <td>
          -
        </td>
        <td>
          10:00-12:00
        </td>
        <td>
          10:00:12-15
        </td>
      </tr>
      <tr
        className="container text-center"
      >
        <td
          colSpan="1"
        >
          Afternoon
        </td>
        <td>
          -
        </td>
        <td>
          -
        </td>
        <td>
          12:45-18:15
        </td>
        <td>
          12:40-17:00
        </td>
        <td>
          -
        </td>
        <td>
          12:30-17:00
        </td>
        <td>
          12:15-16:15
        </td>
      </tr>
      <tr
        className="container text-center"
      >
        <td
          colSpan="1"
        >
          Evening
        </td>
        <td>
          17:25-22:00
        </td>
        <td>
          18:30-22:30
        </td>
        <td>
          18:15-00:15
        </td>
        <td>
          17:30-23:45
        </td>
        <td>
          18:00-00:00
        </td>
        <td>
          17:00-23:30
        </td>
        <td>
          16:45-21:45
        </td>
      </tr>
    </tbody>
  </table>,
]
`;
